scripts/pytest-image-runner.sh

```diff
@@ -50,7 +51,7 @@ build_pytest_image() {
         # NOTE: credentials in both these profiles rotate automatically
 
         # copy web-identity-token file for use inside test container
-        cp "$AWS_WEB_IDENTITY_TOKEN_FILE" "$SCRIPTS_DIR"/web-identity-token >/dev/null
+        cp "$identity_file" "$SCRIPTS_DIR"/web-identity-token >/dev/null
```

Comment:
Above, you are defining TEST_CONTAINER_WEB_IDENTITY_TOKEN_FILE hard-coded to `/root/web-identity-token`. Any reason this is not using `$TEST_CONTAINER_WEB_IDENTITY_TOKEN_FILE` as the destination location for the copy?

Reply:
That env var defines the location of the identity token file within the Pytest container. This line copies the identity file from its location within the Prow container into the temporary directory we use when building the Pytest container. I know that's a bit confusing, but essentially we have to keep track of the file on the host (the Prow container) and the container (the Pytest container) both during build- and then run-time.

Reply:
OK, makes sense. Might be good to add the above as a comment in the file somewhere :)